flow/platforms/ihp-sg13g2/config.mk

```diff
@@ -0,0 +1,132 @@
+# Process node
+export PROCESS = ihp-sg13g2
+
+#-----------------------------------------------------
+# Tech/Libs
+# ----------------------------------------------------
+export TECH_LEF = $(PLATFORM_DIR)/lef/sg13g2_tech.lef
+export SC_LEF = $(PLATFORM_DIR)/lef/sg13g2_stdcell.lef
+
+export LIB_FILES = $(PLATFORM_DIR)/lib/sg13g2_stdcell_typ_1p20V_25C.lib \
+	                  $(ADDITIONAL_LIBS)
+export GDS_FILES = $(PLATFORM_DIR)/gds/sg13g2_stdcell.gds \
+	                  $(ADDITIONAL_GDS)
+#                     $(ADDITIONAL_GDS)
+
+# Dont use cells to ease congestion
+# Specify at least one filler cell if none
+
+# The *probe* are for inserting probe points and have metal shapes
+# on all layers.
+# *lpflow* cells are for multi-power domains
+export DONT_USE_CELLS += \
+sg13g2_antennanp \
+sg13g2_decap_4 \
+sg13g2_decap_8 \
+sg13g2_lgcp_1 \
+sg13g2_sighold \
+sg13g2_slgcp_1 \
+sg13g2_dfrbp_2 \
+sg13g2_fill_1 \
+sg13g2_fill_2 \
+sg13g2_fill_4 \
+sg13g2_fill_8 
```

Comment:
why do fill & decap cells need to be in the dont use list?

Reply:
removed from DONT_USE_CELLS list